### PR TITLE
Update README.md

### LINKED ISSUE #30: Update config.toml
Hey

The readme doesn't reflect the recent changes to config.toml yet.
I have a branch ready with some updates.

I'm loving the theme so far!
Looking forward to tags and categories.
Check out what I made so far at reijn.xyz.

### PR DESCRIPTION
**Pull request by Reijnn**
Fixes #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,19 +21,34 @@ git clone https://github.com/luizdepra/hugo-coder.git themes/coder
 Add the following lines to your `config.toml`.
 
 ```toml
-theme = "coder" # set the theme
```
Comment (khos2ow) [inline README.md]: I would suggest to keep the comment in README file as they are descriptive.

Comment (khos2ow) [reply README.md]: Other than that it LGTM.

Comment (luizdepra) [reply README.md]: Agreed.

**Comment by luizdepra** [comment]
Also, add your name and link to your profile at `Contributors` section.

**Comment by Reijnn** [comment]
Added comments from the official documentation!

**Pull request merged**